libraries/math/src/vector3/coreHelpers.ts

```diff
@@ -75,6 +75,15 @@ export class Vector3Utils {
         return Math.sqrt(v.x ** 2 + v.y ** 2 + v.z ** 2);
     }
 
+    /**
+     * distance
+     *
+     * Calculate the distance between two vectors
+     */
+    static distance(a: Vector3, b: Vector3): number {
```

Comment:
I seem to get errors when I add length, forgot to ping you about that. 
>  guess length is reserved and you can't define a function with that name =S
> Weird though because this is a class
![image](https://github.com/Mojang/minecraft-scripting-libraries/assets/103449886/2abb1365-b9bb-43e7-be42-eb2d9e90fa5a)
Any ideas why that is happening? When I go to definition on length it takes me to function.length which is a property on function, unrelated to the class

Reply:
Ohh and adding distance was just an addition that I thought would be quite useful